samples/build-netlify/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Build on Netlify</title>
+  <script src="https://unpkg.com/contentful-ui-extensions-sdk@3"></script>
+  <link rel="stylesheet" href="https://unpkg.com/contentful-ui-extensions-sdk@3/dist/cf-extension.css" />
+</head>
+
+<body style="margin: 0;">
+  <div id="root" style="display:none">
+    <button id="triggerBuild" class="cf-btn-primary">Build on Netlify</button>
+    <div id="infoBox" style="margin-top: 10px"></div>
+    <ul id="history" style="margin-top: 10px; padding-left: 15px;"></ul>
+  </div>
+  <script>
+
+    function showExtension(params) {
+      document.getElementById('root').style.display = 'block';
+
+      const POLLING_TIME = 3000;
+      const netlifyBuildHookURL = params.instance.netlifyBuildHookURL;
+      const netlifyAPIToken = params.instance.netlifyAPIToken;
```

Comment:
PAT is a common abbreviation. I'd just name this var and extension param `netlifyPAT`